presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -158,6 +158,12 @@ Array Functions
                                        -1,
                                        IF(cardinality(x) = cardinality(y), 0, 1))); -- [[1, 2], [2, 3, 1], [4, 2, 1, 4]]
 
+        SELECT array_sort(ARRAY[ARRAY[1, null], ARRAY[1, 2]]);
+
+        Note : Presto does not support comparison of nested arrays that contain null elements.
```

Comment:
Suggest formatting the indentation of the Note so that the Note is text and not part of the code block, but is still included in the entry for "array_sort(array(T), function(T, T, int)) -> array(T)". 

Screenshot of local build shows how the Note's inclusion in the code block causes highlighting and emphasis that may not be your intention. 

<img width="958" alt="Screenshot 2023-09-29 at 11 12 40 AM" src="https://github.com/prestodb/presto/assets/7013443/e34a8f0b-285e-48a6-a1d4-0d78c1f81f7d">